src/modules/sub-items/sub.items.module.js

```diff
@@ -74,14 +105,23 @@ export default class SubItemsModule extends Component {
         this.setState((state) => ({ items: [...state.items, ...props.items] }));
     }
 
-    /**
-     * Increases an amount of items to be loaded in the list
-     *
-     * @method handleLoadMore
-     * @memberof SubItemsModule
-     */
-    handleLoadMore() {
-        this.setState((state) => ({ offset: state.offset + this.props.limit }));
+    extractFirstSortClauseData(sortClauses) {
```

Comment:
I prefer `extractFirstSortClauseData` over `getDefaultSortClause` because although maybe it is a little bit uglier, I know what it does inside. If you remove context from it, function name still makes sense. This is how I reason about this.

Reply:
I prefer naming methods/functions in the way that I know what it does and what's its purpose. You need this method to get the sort clause when initializing the component, hence why I suggested the name `getDefaultSortClause`. Because when you're initializing something, you'll probably want something to have a default value, like in your code.
If the method was used in many different places, in many different contexts then your reasoning might be correct. Otherwise I prefer my proposal.

Reply:
I second @sunpietro. The method returns the default sort clause.
If for some reason in initial state you would need to have last sort clause, you would like to introduce the new method `extractLastSortClause`?

Reply:
For me both trains of thought are OK, and since we have 2:1 I will change this to `getDefaultSortClause`. :)